sdk/servicebus/service-bus/src/diagnostics/tracing.ts

```diff
@@ -22,12 +27,15 @@ export const tracingClient = createTracingClient({
  */
 export function toSpanOptions(
   serviceBusConfig: Pick<ConnectionConfig, "host"> & { entityPath: string },
+  operation: MessagingOperationNames,
   spanKind?: TracingSpanKind
 ): TracingSpanOptions {
   const spanOptions: TracingSpanOptions = {
     spanAttributes: {
-      "message_bus.destination": serviceBusConfig.entityPath,
-      "peer.address": serviceBusConfig.host,
+      "messaging.system": "servicebus",
+      "messaging.source.name": serviceBusConfig.entityPath,
```

Comment:
this should be conditional depending on operation. `process` and `receive` should use `messaging.source.name`, `publish` should use `messaging.destination.name`

Reply:
Done